src/Parser/MistweaverMonk/CombatLogParser.js

```diff
@@ -0,0 +1,166 @@
+import TalentsTab from 'Main/TalentsTab';
+import CastEfficiencyTab from 'Main/CastEfficiencyTab';
+import CooldownsTab from 'Main/CooldownsTab';
+import ManaTab from 'Main/ManaTab';
+
+
+
+
+import CPM_ABILITIES, { SPELL_CATEGORY } from './CPM_ABILITIES';
+
+function formatThousands(number) {
+  return (Math.round(number || 0) + '').replace(/(\d)(?=(\d{3})+(?!\d))/g, "$1,");
+}
+function formatNumber(number) {
+  if (number > 1000000) {
+    return `${(number / 1000000).toFixed(2)}m`;
+  }
+  if (number > 10000) {
+    return `${Math.round(number / 1000)}k`;
+  }
+  return formatThousands(number);
+}
+function getIssueImportance(value, regular, major, higherIsWorse = false) {
+  if (higherIsWorse ? value > major : value < major) {
+    return ISSUE_IMPORTANCE.MAJOR;
+  }
+  if (higherIsWorse ? value > regular : value < regular) {
+    return ISSUE_IMPORTANCE.REGULAR;
+  }
+  return ISSUE_IMPORTANCE.MINOR;
+}
+function formatPercentage(percentage) {
+  return (Math.round((percentage || 0) * 10000) / 100).toFixed(2);
+}
+
+class CombatLogParser extends MainCombatLogParser {
+  static specModules = {
+    // Override the ability tracker so we also get stats for IoL and beacon healing
+    // abilityTracker: AbilityTracker,
+
+    // Features
+    // alwaysBeCasting: AlwaysBeCasting,
+    // cooldownTracker: CooldownTracker,
+
+
+    // Legendaries:
+    // drapeOfShame: DrapeOfShame,
+    // velens: Velens,
+    // prydaz: Prydaz,
+    // sephuz: Sephuz,
+
+
+    // Shared:
+    //amalgamsSeventhSpine: AmalgamsSeventhSpine,
+    //darkmoonDeckPromises: DarkmoonDeckPromises,
+  };
+
+  generateResults() {
+    const results = new ParseResults();
+
+
+    results.statistics = [
+      <StatisticBox
+        icon={(
+          <img
+            src="./img/healing.png"
+            style={{ border: 0 }}
+            alt="Healing"
+          />)}
+        value={`${formatNumber(this.totalHealing / this.fightDuration * 1000)} HPS`}
+        label={(
+          <dfn data-tip={`The total healing done recorded was ${formatThousands(this.totalHealing)}.`}>
+            Healing done
+          </dfn>
+        )}
+      />,
+    ];
+
+    results.items = [
+      /*this.modules.prydaz.active && {
+        id: ITEMS.PRYDAZ_XAVARICS_MAGNUM_OPUS.id,
+        icon: <ItemIcon id={ITEMS.PRYDAZ_XAVARICS_MAGNUM_OPUS.id} />,
+        title: <ItemLink id={ITEMS.PRYDAZ_XAVARICS_MAGNUM_OPUS.id} />,
+        result: (
+          <dfn data-tip="The actual effective healing contributed by the Prydaz, Xavaric's Magnum Opus equip effect.">
+            {((prydazHealingPercentage * 100) || 0).toFixed(2)} % / {formatNumber(this.modules.prydaz.healing / fightDuration * 1000)} HPS
+          </dfn>
+        ),
+      },*/
+    ];
+
+    results.tabs = [
+      {
+        title: 'Suggestions',
+        url: 'suggestions',
+        render: () => (
+          <SuggestionsTab issues={results.issues} />
+        ),
+      },
+    /*  {
```

Comment:
It would be nice to have all these commented out tabs from the start. Without them I think the added value of the analyzer is too small.